build/x86_64_centos7/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
+
+ENV PATH ${PATH}:/usr/local/bin
+
+RUN yum update -y && yum -y install \
+  autoconf \
```

Comment:
This is the last thing that stands out to me. Can we keep the indentation the same as what we have for debian specifically for the packages that are being installed? They should be one level deeper than the rest of the chained commands.

Reply:
Done.